axelor-human-resource/src/main/java/com/axelor/apps/hr/web/expense/ExpenseController.java

```diff
@@ -683,13 +672,13 @@ public void computeDistanceAndKilometricExpense(ActionRequest request, ActionRes
         expense = context.getParent().asType(Expense.class);
       }
 
-      Employee employee = expense.getUser().getEmployee();
+      Employee employee = expense.getEmployee();
 
       if (employee == null) {
         throw new AxelorException(
             TraceBackRepository.CATEGORY_CONFIGURATION_ERROR,
             I18n.get(IExceptionMessage.LEAVE_USER_EMPLOYEE),
-            expense.getUser().getName());
+            AuthUtils.getUser().getName());
       }
```

Comment:
please remove this check as employee cannot be null here and the message is wrong

Reply:
Here need to keep this check as employee is required on view side. So, it can be null